src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -324,6 +325,90 @@ void jit_exp_emitter::register_table_entries() {
     push_arg_entry_of("exponent_bias", 0x0000007f);
 }
 
+/// FLOOR ///
+jit_floor_emitter::jit_floor_emitter(jit_generator* host, cpu_isa_t host_isa, const element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {
+        prepare_table();
+    }
+
+jit_floor_emitter::jit_floor_emitter(ov::intel_cpu::riscv64::jit_generator* host, ov::intel_cpu::riscv64::cpu_isa_t host_isa, const std::shared_ptr<ov::Node>& node) :
+    jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {
+        prepare_table();
+}
+
+size_t jit_floor_emitter::get_inputs_num() const {
+    return 1;
+}
+
+size_t jit_floor_emitter::aux_vecs_count() const {
+    return 2; 
+}
+
+size_t jit_floor_emitter::aux_fp_gprs_count() const {
+    return 2; 
+}
+
+void jit_floor_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == ov::intel_cpu::riscv64::cpu_isa_t::gv) {
+        emit_isa<ov::intel_cpu::riscv64::cpu_isa_t::gv>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel for FLOOR");
+    }
+}
+void jit_floor_emitter::register_table_entries() {
+
+    push_arg_entry_of("zero", 0x00000000);    
+    push_arg_entry_of("neg_one", 0xbf800000);
+}
+
+template <ov::intel_cpu::riscv64::cpu_isa_t isa>
+void jit_floor_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const {
+    VReg src = VReg(in_vec_idxs[0]);
+    VReg dst = VReg(out_vec_idxs[0]);
+    VReg aux1 = VReg(aux_vec_idxs[0]);    
+    VReg aux2 = VReg(aux_vec_idxs[1]); 
+    FReg fp0 = FReg(aux_fp_gpr_idxs[0]);
+    FReg fp1 = FReg(aux_fp_gpr_idxs[1]);
+
+
+    h->vmv_v_v(aux1, src);                
+    load_table_val("zero", fp0);               
+    h->vmflt_vf(mask_vreg(), src, fp0);      
+    h->vmv1r_v(aux2, mask_vreg());           
+
+    if (exec_prc_ == element::f32) {
+        
+        h->vfcvt_x_f_v(dst, src);            
+        h->vfcvt_f_x_v(dst, dst);            
+
+        h->vmv1r_v(mask_vreg(), aux2);       
```

Comment:
I see that we copy values from `aux2` to `mask` here and... rewrite mask on the next line `h->vmfgt_vv(mask_vreg(), dst, aux1);`.

As far as I understand, `aux2` contains mask for negative values (less zero) due to he lines L374-377. But the code looks so that we don't need to this mask and `aux2` - we don't operate with negative values in the algorithm.

Because of that may I ask you please to remove lines L374-377 and L384 (lines which work with `aux2`)? Then it seems that we need only one vector aux register  and the method `aux_vecs_count()` should return 1.